yamls/distributed_ppo_test.yaml

```diff
@@ -12,13 +12,208 @@ integrations:
 - integration_type: git_repo
   path: /workspace/compose-rl
   git_repo: databricks/compose-rl
-  git_branch: single-controller-hackathon
+  git_branch: ethantang-db/orig_yaml
+- integration_type: git_repo
+  path: /workspace/research-universe
+  git_repo: databricks-mosaic/research-universe
+  ssh_clone: true
+  git_branch: main
```

Comment:
Probably best to use the `update-orl-eval` branch on `research-universe` or your code might fail